compiler/resolution/functionResolution.cpp

```diff
@@ -4720,7 +4720,6 @@ static bool
 isNormalField(Symbol* field)
```

Comment:
Given this name and the vagueness of the term 'normal' (e.g., is a 'type' normal?  a 'param'?), it might be kind while modifying its definition to add a short comment to this routine explaining what we think it does now.  (E.g., I'm guessing it's something like "returns true for user fields including const, param, and type fields; false for things like the compiler-introduced 'super' and 'outer' fields).